packages/oui-skeleton/skeleton.less

```diff
@@ -38,3 +38,7 @@
     }
   }
 }
+
+oui-skeleton {
```

Comment:
How did it work before ? 🤔 It looks like `oui-skeleton` has `.oui-skeleton` so it looks like the style will be applied twice

Reply:
No, it'll extend the selector above and the selector will catch the same element.
In fact, we could even avoid adding `.oui-skeleton` on the `oui-skeleton` component.
But it's harmless for now. Could be an enhancement for the next major version :)